src/util/logger/Logger.cpp

```diff
@@ -47,8 +47,10 @@ void Logger::log(std::string message, const std::string log_type) {
     long millis = tv.tv_sec * 1000 + tv.tv_usec / 1000;
     // TODO: This temporarily fixes spdlog hanging after forking. This will prevent using the actual logger and simulate
     // the behavior using cout instead. But it will not write logs to the log file.
-    cout << " [" << millis << "] [" << log_type << "] [" << worker_name << " : " << getpid() << ":" << tid << "] "
-         << message << endl;
+    if (log_type.compare("debug") != 0) {
```

Comment:
If its other log type other than debug what will happen here?

Reply:
This if statement prevent debug logs only. So other logs printed as usual way.

Reply:
Ok